tests/stress.rs

```diff
@@ -4677,18 +4677,20 @@ unsafe extern "C" fn stress_functions(context: &dc_context_t) {
     };
 }
 
-unsafe extern "C" fn cb(
-    _context: &dc_context_t,
-    _event: Event,
-    _data1: uintptr_t,
-    _data2: uintptr_t,
-) -> uintptr_t {
-    0
-}
+#[cfg(test)]
```

Comment:
you don't need this because the file is in `tests` so it is already in the tests module

Reply:
the splitting idea is to have a file like this: https://github.com/rpgp/rpgp/blob/master/tests/key_test.rs

Reply:
> you don't need this because the file is in tests so it is already in the tests module

Makes sense. Guess I don't need the `mod tests {}` block either